AlfrescoApp/Model/Managers/RealmSyncManager+CoreDataMigration.m

```diff
@@ -32,6 +32,106 @@
 
 @implementation RealmSyncManager (CoreDataMigration)
 
+- (BOOL)isContentMigrationNeeded
+{
+    NSURL *sharedAppGroupFolderURL = [[NSFileManager defaultManager] containerURLForSecurityApplicationGroupIdentifier:kSharedAppGroupIdentifier];
+    NSString *documentsDirectoryPath = [NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES) lastObject];
```

Comment:
These two variables are not used at all in the current method. Is there something missing or can they be removed?